yb-voyager/src/migassessment/assessmentDB.go

```diff
@@ -199,27 +210,33 @@ const (
 		tii.seq_reads AS reads,
 		tii.row_writes AS writes,
 		0 AS is_index,
+		otm.object_type,
 		NULL AS parent_table_name,
 		tis.size_in_bytes
 	FROM %s trc
 	LEFT JOIN %s tii ON trc.schema_name = tii.schema_name AND trc.table_name = tii.object_name and tii.measurement_type='initial'
 	LEFT JOIN %s tis ON trc.schema_name = tis.schema_name AND trc.table_name = tis.object_name
-	LEFT JOIN %s tcc ON trc.schema_name = tcc.schema_name AND trc.table_name = tcc.object_name;`
+	LEFT JOIN %s tcc ON trc.schema_name = tcc.schema_name AND trc.table_name = tcc.object_name
+	LEFT JOIN %s otm ON trc.schema_name = otm.schema_name AND trc.table_name = otm.object_name
+	WHERE otm.object_type NOT IN ('%s', '%s');`
```

Comment:
Pls add a comment explaining why we're excluding partitioned tables and indexes

Reply:
Sure I will add the comment, here is the reason:
```
For partitioned tables, only the leaf partitions have the actual data, and read/write IOPS information.

In PG
1. For non-leaf partitions, table type is `Partitioned Table` and Indexes type is `Partitioned Index`
2. For leaf partitions, table type is `table` and Indexes type is `index`

The Sizer algorithm can only focus on analyzing and recommending using the leaf partitions only
```